includes/wikia/services/UserStatsService.class.php

```diff
@@ -124,54 +101,49 @@ public function getEditCountWiki( $wikiId = 0, $skipCache = false, $flags = 0 )
 	 *
 	 * @return Int
 	 */
-	public function getEditCountGlobal( $wikiId = 0, $skipCache = false ) {
+	public function getEditCountGlobal( $skipCache = false ) {
```

Comment:
This function serves same purpose as \User::getEditCount I'd consider removing it.